src/utils/log.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+var chalk = require('chalk');
+var moment = require('moment');
+
+var red = chalk.red;
+var green = chalk.green;
+var blue = chalk.blue;
+var yellow = chalk.yellow;
+var hour = chalk.grey.bold;
+
+var log = function (serverName, middlewareName, method, from, to, status) {
```

Comment:
With so many arguments, maybe is better to use a config object as single argument